setup.cfg

```diff
@@ -38,25 +38,26 @@ format = pylint
 statistics = 1
 show-source = 1
 exclude =
-    build,
-    .git,
     .circleci,
     .coverage,
     .eggs,
+    .git,
     .gitignore,
     .idea,
-    coverage,
-    dist,
-    *.egg*,
-    venv,
-    venv_py3,
     .tox,
+    *.egg*,
     __pycache__
+    build,
+    coverage,
+    dist,
+    venv*,
 inline-quotes = "
 multiline-quotes = "
 accept-encodings = utf-8
 import-order-style = pep8
 
+# Note(brianm): this is deprecated.
```

Comment:
Just a nit - prefer to track things like this in issues rather than docs spread in codebase, that way they're easier to find + track.

Reply:
Word. Migrated to issue: https://github.com/spotify/spotify-tensorflow/issues/119